# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../colordiff/git-diff-colors.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

We shouldn't apply this kind of patches in generic, but the
default colors are... way too awful. I'll use something similar
to `git diff --color`

--- ./colordiffrc.orig	2008-07-03 15:25:47.000000000 +0000
+++ ./colordiffrc	2008-07-03 15:26:05.000000000 +0000
@@ -20,7 +20,7 @@
 # this, use the default output colour"
 #
 plain=off
-newtext=blue
-oldtext=red
-diffstuff=magenta
-cvsstuff=green
+newtext=darkgreen
+oldtext=darkred
+diffstuff=darkyellow
+cvsstuff=white
